pkg/reconciler/broker/trigger/controller.go

```diff
@@ -163,7 +163,15 @@ func filterTriggers(lister eventinglisters.BrokerLister) func(interface{}) bool
 			return false
 		}
 
-		b, err := lister.Brokers(trigger.Namespace).Get(trigger.Spec.Broker)
+		if trigger.Spec.BrokerRef != nil {
```

Comment:
I can't check the feature flag here because I don't have the ctx?

Reply:
I would add another parameter to the function (and maybe the one that calls it as well if needed) to pass the ctx to here

Reply:
at this level, you have access to the feature store directly

Reply:
@pierDipi @Cali0707 Would the feature store be passed in as a parameter as well?